Kickstarter-iOS/Views/Controllers/RewardsCollectionViewController.swift

```diff
@@ -25,7 +25,7 @@ final class RewardsCollectionViewController: UICollectionViewController {
     UICollectionViewFlowLayout()
       |> \.minimumLineSpacing .~ Styles.grid(3)
       |> \.minimumInteritemSpacing .~ 0
-      |> \.sectionInset .~ .init(topBottom: Styles.grid(6))
+      |> \.sectionInset .~ .zero
```

Comment:
Do we need to set this explicitly? This is what docs say about `sectionInset`: `The default edge insets are all set to 0.`